### DISCUSSION TITLE
Percy/cli 1.20.0 domSnapshot: missing required property.

### DISCUSSION THREAD

**Discussion by slokerqe** [percy/cli#1188] [General]
@percy/cli -> **1.20.0**
receiving this message:
2023-02-21T05:53:24.3702098Z [percy] Invalid snapshot options:
2023-02-21T05:53:24.3702791Z [percy] - domSnapshot: missing required property
2023-02-21T05:53:25.1644208Z [percy] Snapshot taken: <my_custom_snapshot_name>

*After execution, Percy's build includes empty snapshots
Please Note, I've downgraded to 1.18.0 and all works as expected

- Java 11
- gradle 8.0
- node -v --> v18.14.1
- build.gradle:
  plugins {
    id 'java'
}

sourceCompatibility = 1.11
targetCompatibility = 1.11

group 'org.example'
version '1.0-SNAPSHOT'

repositories {
    mavenCentral()
}

dependencies {
    testImplementation group: 'io.percy', name: 'percy-java-selenium', version: '1.2.0'
    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter', version: '5.9.1'
    testImplementation group: 'com.codeborne', name: 'selenide', version: '6.11.2'
    testImplementation group: 'org.slf4j', name: 'slf4j-simple', version: '2.0.6'
}

tasks.withType(Test).configureEach {
    systemProperty('selenide.browser', 'chrome')
    systemProperty('selenide.fastSetValue', 'true')
    systemProperty('selenide.browserSize', '1920x1080')

    systemProperties += System.properties

    useJUnitPlatform {
        includeEngines 'junit-jupiter'
    }

    reports {
        junitXml.mergeReruns = true
    }

    testLogging.showStandardStreams = true
    testLogging.exceptionFormat = 'full'
}

- package.json:
{
  "private": true,
  "devDependencies": {
    "@percy/cli": "1.20.0"
  }
}

- percy snapshot command:
new Percy(WebDriverRunner.getWebDriver()).snapshot(snapshotName);

- percy yml:
version: 2
snapshot:
  percy-css: |
    input[formcontrolname="***"] {
      display: none;
    }
discovery:
  disable-cache: true
upload:
  files: "**/*.{png,jpg,jpeg}"
  ignore: ""
  stripExtensions: false
  
  
- percy exec command:
npx percy exec -- ./gradlew clean {name of suite}

That is how snapshot looks like
![image](https://user-images.githubusercontent.com/11923504/220350159-7d8c8a0e-a9c5-4b8e-b1ec-ced7a9510b17.png)

**Comment by itsjwala** [comment]
@slokerqe can you share full unedited logs as well?

**Reply by itsjwala** [reply]
* I think it would be failing on [1.19.2](https://github.com/percy/cli/releases/tag/v1.19.2) as well for you, since there is nothing in `1.20.0` that would cause this issue.
* Please send a reproducible example/test project for us to debug this further.

**Reply by slokerqe** [reply]
I found out that this is not working only with my project.
Due to security reasons, I'll not post more information here.
Hoping it will be fixed somehow. Thank you!

**Reply by kpamnani** [reply]
Is there any resolution for this issue?

Facing this even with @percy/cli@1.27.7 and
        <dependency>
            <groupId>io.percy</groupId>
            <artifactId>percy-java-selenium</artifactId>
            <version>2.0.2</version>
        </dependency>
